DOL.WHD.Section14c.Web/src/modules/components/sectionWageData/sectionWageDataController.test.js

```diff
@@ -0,0 +1,19 @@
+describe('sectionWageDataController', function() {
+
+    beforeEach(module('14c'));
+
+    beforeEach(inject(function ($rootScope, $controller) {
+        scope = $rootScope.$new();
+
+        sectionWageDataController = function() {
+            return $controller('sectionWageDataController', {
+                '$scope': scope, 
+                '$route': route
+            });
+        };
+    }));
+
+    it('invoke controller', function() {
+        var controller = sectionWageDataController();
```

Comment:
added valid test and todo to cover remaining code in controller https://github.com/AppliedIS/dol-whd-14c/commit/451dd1617913cee68c2fdcb7945d4b7598a5be93